src/components/SecondaryPanes/CommandBar.css

```diff
@@ -2,6 +2,7 @@
   flex: 0 0 29px;
   border-bottom: 1px solid var(--theme-splitter-color);
   display: flex;
+  justify-content: flex-end;
```

Comment:
this also moves all of the stepping buttons over to the right hand side of the bar, which is not something we want to do right now. lets remove this rule